uber/models/__init__.py

```diff
@@ -737,6 +755,30 @@ def process_refund(self, stripe_log, model=Attendee):
 
                 return '', response
 
+        def add_receipt_items_by_model(self, charge, model):
+            for amount, desc, item_type in getattr(model, 'receipt_items_breakdown'):
+                if amount:
+                    if "Promo Code Group" in desc:
```

Comment:
This code won't actually trigger with the current state of the code, as the corresponding receipt item property will never return true. This is because of how we handle pre-reg promo code groups (i.e., badly). That all being said, this will come in handy if we add the ability to buy a group of promo code badges after you've pre-registered.